docs/dcp2_system_design.rst

```diff
@@ -594,7 +596,7 @@ The following regex can be used to validate dataset names (line breaks added
 for legibility)::
 
     ^
-    hca_(dev|prod|staging)
+    (hca|lungmap)_(dev|prod|staging)
```

Comment:
Depending on the answer above we may need to re-write these regex

Reply:
Good catch, updated (atlas is non-optional)